service/build.gradle

```diff
@@ -37,6 +37,7 @@ dependencies {
 
 	implementation 'com.heroku.sdk:env-keystore:1.1.6'
 
+    implementation 'junit:junit:4.12'
```

Comment:
junit should not be an 'implementation' dependency; and is not needed anyway as spring-boot-starter-test includes junit5